tests/kernel/semaphore/sys_sem/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_ZTEST=y
+CONFIG_TEST_USERSPACE=y
+CONFIG_MP_NUM_CPUS=1
```

Comment:
Generally it would be nice for new tests to be SMP-safe.  I don't see anything here that strictly requires single-cpu operation to test what it's needs to exercise...

Reply:
Oh, I forgot, I will request run testcases on single cpu. Thanks

Reply:
Thanks, I deleted this configuration.